tests/openbis_functions/test_openbis_functs.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from lebedigital.openbis.expstep import ExpStep
+
+
+@pytest.mark.skip(reason='The directory for the yaml file does not exist in tests workflow. Test is dependent on MinimumWorkingExample')
+def test_load_emodul():
+    """Testing if the metadata from yaml emodul files are imported corretly
+    """
+
+    yaml_path = '../usecases/MinimumWorkingExample/emodul/metadata_yaml_files/Wolf 8.2 Probe 1.yaml'
+
+    test_sample = ExpStep(name='test_sample')
```

Comment:
I have already written a test object generator, i will use it as a pytest fixture. It could then take on being the test subject for the tests i would want to write. We would only need to modify the fixture then.